packages/studio-ui/src/web/views/FlowBuilder/nodeProps/StandardNode.jsx

```diff
@@ -12,9 +12,11 @@ const style = require('./style.scss')
 
 export default class StandardNodePropertiesPanel extends Component {
   renameNode = text => {
-    if (text) {
+    if (text !== this.props.node.name) {
```

Comment:
This brings another issue. An empty name will create a node where all the other nodes will point to
![image](https://user-images.githubusercontent.com/42552874/139745175-a8967355-7943-4f4a-a03b-9cc31dcf1120.png)

Also, please rename the method, "renameNode" is not accurate, it's "onChange". So we need to check if text is set and is not blank

Reply:
You are right my bad !